src/index.js

```diff
@@ -8,5 +8,6 @@ export { default as getQueryKey } from './lib/get-query-key';
 export { default as queriesReducer } from './reducers/queries';
 export { default as entitiesReducer } from './reducers/entities';
 export { default as queryMiddleware } from './middleware/query';
+export { default as queryMiddlewareAdvanced } from './middleware/query-advanced';
```

Comment:
Additionally I think we can modify the webpack config to export a new UMD build for `redux-query/advanced`.